Controller/ConnectController.php

```diff
@@ -387,11 +387,7 @@ private function getConfirmationResponse(Request $request, array $accessToken, s
         $event = new FilterUserResponseEvent($currentUser, $request, $response);
         $this->dispatch($event, HWIOAuthEvents::CONNECT_COMPLETED);
 
-        if (null !== $event->getResponse()) {
-            return $event->getResponse();
-        }
-
-        return $response;
+        return $event->getResponse() ?: $response;
```

Comment:
```suggestion
        return $event->getResponse() ?? $response;
```

I think expresses better that `$event->getResponse()` can only be `null` or `Response`.

Reply:
As we bumped to PHP 7.4 recently, forced type & return type to not be nullable.